### ISSUE TITLE
Image not clear

### ISSUE THREAD

**Issue by aravindvnair99** [ishandeveloper/ishandeveloper#3]
Just thought I'll let you know...

# Dark mode (Not readable and not clear):

![image](https://user-images.githubusercontent.com/22199259/116465358-c3de2180-a88a-11eb-81f2-e5ee974bc200.png)

# Light mode (Much more readable and clearer):

![image](https://user-images.githubusercontent.com/22199259/116465511-f38d2980-a88a-11eb-8351-2bd7dcca6d6a.png)

**Comment by ishandeveloper** [comment]
Thanks for reporting @aravindvnair99. This is a known issue and I'm trying to figure out a workaround for this.

Basically I'm using a white border while rendering the svg, so that it's invisible in github's light mode theme, while still supporting github's dark theme at the same time.

Let me know if you have any suggestions on how we can improve this.

**Comment by aravindvnair99** [comment]
@ishandeveloper I think writing that in text format would be the best as GitHub automatically switches colour for text. That also gives you the choice of hyperlinking to your profiles. Text can be used with SVG icons.

**Comment by ishandeveloper** [comment]
@aravindvnair99 Yes, but it'd be difficult to achieve a similar layout using markdown and I'm planning to release this readme svg renderer as an open-sourced api later on, so that other people can also create similar readme headers with ease, it'd be a bad ux if we ask everyone to copy-paste a 50-line markdown code to achieve this, rather than linking an image directly.
